go.mod

```diff
@@ -3,11 +3,11 @@ module github.com/giantswarm/kvm-operator
 go 1.14
 
 require (
-	github.com/giantswarm/apiextensions v0.3.9
+	github.com/giantswarm/apiextensions v0.3.10-0.20200513205053-0bf9395d65eb
 	github.com/giantswarm/certs v0.2.0
 	github.com/giantswarm/errors v0.2.3
 	github.com/giantswarm/k8sclient v0.2.0
-	github.com/giantswarm/k8scloudconfig/v6 v6.1.1
+	github.com/giantswarm/k8scloudconfig/v6 v6.1.2-0.20200518162712-4229933edecf
```

Comment:
This will be changed to 6.1.2 when https://github.com/giantswarm/k8scloudconfig/pull/708 is released.